pom.xml

```diff
@@ -64,7 +64,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.enforcer.requireJavaVersion>${maven.compiler.source}</maven.enforcer.requireJavaVersion>
-        <liquibase.version>4.7.1</liquibase.version>
+        <liquibase.version>0-SNAPSHOT</liquibase.version>
```

Comment:
Note: need to get https://github.com/liquibase/liquibase/pull/2569/ merged and released first